Gemfile.lock

```diff
@@ -75,7 +75,7 @@ GEM
       middleman (~> 3.0)
       middleman-sprockets (~> 3.0)
       sass
-    ember-source (2.4.1)
+    ember-source (2.9.0)
```

Comment:
@rwjblue from talking to @kiwiupover earlier today it sounds like things broke badly with 2.12 ...